Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -248,11 +251,42 @@ static VertexLoaderBase* RefreshLoader(int vtx_attr_group, bool preprocess = fal
   return loader;
 }
 
-int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bool is_preprocess)
+void UpdateCyclesPerVertex()
 {
-  if (!count)
-    return 0;
+  // Note: These timings do not consider the post-transform vertex cache. These counts are verified
+  // on hardware based on the result of GX_ReadClksPerVtx().
```

Comment:
Were these timings measured with direct or indirect vertex data?

I assume using vertex buffers has some impact on timing. 

Also, would vertices with a large amount of data (direct or indirect) cost more?